src/utils.ts

```diff
@@ -46,12 +46,14 @@ export function getRemoteDetail(remoteUrl: string): RemoteDetail {
     throw new Error('Could not parse GitHub Server name')
   }
 
+  remoteUrl = remoteUrl.replace(/\.git$/, '')
+
   const httpsUrlPattern = new RegExp(
     '^https?://.*@?' + githubServerMatch[1] + '/(.+/.+)$',
```

Comment:
I would prefer to fix this by only changing the regex.
```typescript
'^https?://.*@?' + githubServerMatch[1] + '/(.+/.+?)(.git)?$',
```

Also, please add a test for this pattern here:
https://github.com/peter-evans/create-pull-request/blob/main/__test__/utils.unit.test.ts#L35-L53

Reply:
> I would prefer to fix this by only changing the regex.
👍